src/api/movie_api.ts

```diff
@@ -0,0 +1,145 @@
+import {Filters} from 'src/utils/filters'
+import axios from 'axios'
+import {useInfiniteQuery, UseInfiniteQueryResult, useQuery} from 'react-query'
+
+const apiURL = import.meta.env.VITE_API_URL
+const apiKEY = import.meta.env.VITE_API_KEY
```

Comment:
hmmmm, is this safe? It's the first that I see this.

Reply:
No it's not safe. But the data I'm dealing with is for expermintal puropses so I wouldn't mind.